Content.Client/Weapons/Melee/MeleeWeaponSystem.Effects.cs

```diff
@@ -205,4 +211,27 @@ private Animation GetLungeAnimation(Vector2 direction)
             }
         };
     }
+
+    /// <summary>
+    /// Updates the effect positions to follow the user
+    /// </summary>
+    private void UpdateEffects()
+    {
+        var query = EntityQueryEnumerator<TrackUserComponent, TransformComponent>();
+        while (query.MoveNext( out var arcComponent, out var xform))
+        {
+            if (arcComponent.User == null)
+                continue;
+
+            Vector2 targetPos = TransformSystem.GetWorldPosition(arcComponent.User.Value);
+
+            if (arcComponent.Offset != Vector2.Zero)
+            {
+                var entRotation = xform.LocalRotation;
```

Comment:
This should use the effect's WorldRotation, not local rotation.

Reply:
oop